docker/register.sh

```diff
@@ -121,27 +122,29 @@ setup_operator() {
 
     echo "Using LST_CONTRACT_ADDRESS: $LST_CONTRACT_ADDRESS"
     echo "Using LST_STRATEGY_ADDRESS: $LST_STRATEGY_ADDRESS"
-    
+
     # TODO: is this write? need proper LST addr setup in the .env file
+    # TODO: Need to be able to change the amount we stETH stake relative to their balance. TRight now it hands for me?
+    AMOUNT=10000 # TODO: make this a value set via cli args / env variables to override
```

Comment:
Good idea. But let's make the default higher. Like 0.01ETH at least. (Which would  be a 1 with 16 zeros)

Reply:
set as configurable now when calling `/wavs/register.sh`